<ui version="4.0" >
 <class>DosboxGamesDialog</class>
 <widget class="QDialog" name="DosboxGamesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DOSBox - Games</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/dboxfe-gdb.qrc" >:/images/copy.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxOption" >
     <property name="title" >
      <string>Option</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelDosboxVersion" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>DOSBox Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDosboxVersion" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelSearch" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearch" />
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearch" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBoxGames" >
     <property name="title" >
      <string>Games</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="treeWidgetDosboxGames" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelGames" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Gamename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGames" />
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnAccept" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/dboxfe-gdb.qrc" >:/images/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/dboxfe-gdb.qrc" >:/images/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/dboxfe-gdb.qrc" />
 </resources>
 <connections/>
</ui>
